server/services/snip.js

```diff
@@ -0,0 +1,124 @@
+const snipModel = require('../models/snip');
+const config = require('../config')
+const lockTypes = require('../constants/lockTypes');
+var md5 = require('MD5')
+
+//TODO handle lock
+const findByUrl = async url => {
+    try {
+        return await snipModel.findByUrl(url);
+    } catch (error) {
+        console.error(error);
+        throw "Unable to find snip data";
+    }
+}
+
+//TODO handle lock
+const deleteByUrl = async url => {
+    try {
+        const snip = await snipModel.findByUrl(url);
+        return await snip.remove();
+    } catch (error) {
+        console.error(error);
+        throw "Unable to delete snip";
+    }
+}
+
+// TODO change md5 to sha1
```

Comment:
Nothing to do here!
Its wrt `hashPassword` function

Reply:
need to change md5 to sha1

Reply:
I mean, the TODO should be given for `hashPassword` function definition.